src/ansys/units/quantity_tables/cgs_table.yaml

```diff
@@ -4,95 +4,89 @@
 # Quantity: units
 
 quantity_units_table:
```

Comment:
@hpohekar sorry for the confusion. The table with the Fluent Scheme-style strings is intended only for the present integration with the settings API via its units-quantity attributes. The tables in this library are to remain independent of that ad hoc table.

The only changes required here are to remove unneeded entries, and to add missing ones.

Reply:
@seanpearsonuk No worries at all. We will create new PR to address this after 0.2.1 release. Thank you.